CHANGES.rst

```diff
@@ -71,6 +78,9 @@ astropy.units
 astropy.utils
 ^^^^^^^^^^^^^
 
+- ``ShapedLikeNDArray`` has gained the capability to use numpy functions
```

Comment:
And moved into a new module, so API change?

Reply:
Good point! Done